### ISSUE TITLE
After event/notification expired the data in DB has to delete 

### ISSUE THREAD

**Issue by ajithkpistar** [ajith-kp-1892/Bmsit-Notifier#14]
older events are not suppose to display...this is sql issue....chinmay can u pls look into this also...

**Comment by chinmaybhat1** [comment]
This can be achieved using cron job but it is really a bad practice. So instead of deleting the data we can simply set a flag and disable the notification or event

**Comment by ajith-kp-1892** [comment]
yeah no problem you make a changes in db and tell me and wen  querying data we can achive  this i think.hav u made anything regarding this in the prev merge???
